apps/server/src/modules/oauth/service/oauth.service.spec.ts

```diff
@@ -159,10 +159,22 @@ describe('OAuthService', () => {
 			httpService.post.mockReturnValue(of(createAxiosResponse<OauthTokenResponse>(tokenResponse)));
 		});
 
-		it('should get token from the external server', async () => {
-			const responseToken: OauthTokenResponse = await service.requestToken(code, testOauthConfig);
+		describe('when authorization code and oauthConfig is given', () => {
+			it('should get token from the external server', async () => {
+				const responseToken: OauthTokenResponse = await service.requestToken(code, testOauthConfig);
 
-			expect(responseToken).toStrictEqual(tokenResponse);
+				expect(responseToken).toStrictEqual(tokenResponse);
```

Comment:
I cant get the expection

Reply:
now it should be more useful/meaningful :)